cmd/root.go

```diff
@@ -38,8 +38,8 @@ func AddCommands(cmd *cobra.Command) {
 		config.OracleServer, "The oracle server url. (temporary) ")
 	cmd.PersistentFlags().IntVar(&config.OracleServerWorldID, "oracle_server_worldid",
 		config.OracleServerWorldID, "The worldid to use with the oracle server (temporary) ")
-	cmd.PersistentFlags().StringVar(&config.PoETServer, "poet-server",
-		config.PoETServer, "The poet server url. (temporary) ")
+	cmd.PersistentFlags().StringArrayVar(&config.PoETServers, "poet-server",
+		config.PoETServers, "The poet server url. (temporary) Can be passed multiple times")
```

Comment:
`StringSliceVar` works differently and not how I wanted it to. I want users to pass each PoET separately (like `--poet-server=foo --poet-server=bar`).

Reply:
Both `StringSliceVar` and `StringArrayVar` will parse your example in the same way:`--poet-server=foo --poet-server=bar` will result in `{"foo", "bar"}`

The difference is in the following: `--poet-server=foo,bar`. With `StringArrayVar` this will be `{"foo,bar"}` with `StringSliceVar` this will be `{"foo", "bar"}`. See also https://github.com/spf13/cobra/issues/661#issuecomment-377684634

Reply:
Yes, I know. I don't want to allow passing `--poet-server=foo,bar`.